hack/rpm-deps.sh

```diff
@@ -105,6 +105,28 @@ bazel run \
     $libvirtdevel_base \
     $libvirtdevel_extra
 
+bazel run \
+    --config=${ARCHITECTURE} \
+    //:bazeldnf -- rpmtree \
+    --public --nobest \
+    --name sandboxroot_x86_64 \
+    --repofile rpm/centos-repo.yaml \
+    -f centos-stream-release \
+    $centos_base \
+    $centos_extra \
+    gcc python36
```

Comment:
Can you please follow what is done for all other rpmtrees starting with my patches, that is define

```bash
sandboxroot_base = "
    gcc
    python36
"
```

earlier and reuse it here? That'd remove some duplication and make future patches nicer.

Reply:
Done.